adapter/internal/api/apis_impl.go

```diff
@@ -151,6 +174,31 @@ func extractAPIProject(payload []byte) (apiProject ProjectAPI, err error) {
 	return apiProject, nil
 }
 
+func verifyMandatoryFields(apiJSON config.APIJsonData) error {
+	var errMsg string = ""
+	if apiJSON.Data.APIName == "" {
+		errMsg = "API Name cannot be empty"
+	}
+
+	if apiJSON.Data.APIVersion == "" {
+		errMsg = "API Version cannot be empty"
+	}
+
+	if apiJSON.Data.APIContext == "" {
```

Comment:
If API Name, Version is available shall we add those to the log message as well.
If it is not available, we can proceed with some key word like "unknown"

Reply:
If we proceed we need API Name, Version, Context, endpoint every thing right? Can add debug messages in every if condition